### ISSUE TITLE
Power button not working on Redmi 2 with MIUI 7

### ISSUE THREAD

**Issue by samiraguiar** [SpazeDog/xposed-additions#26]
When the module is enabled, the power button doesn't wake or sleep the device. 
The button is recognized by Xposed Additions but even after remapping it as the power button this won't work. Pressing and holding it to force the device to reboot works fine.

**Comment by dbergloev** [comment]
First try updating to v.3.6.0. It still uses the same remap engine, but the error log monitor has been fixed to work on newer Android versions. If any errors occurs, you will be able to see it in the log viewer. If the viewer is empty, you need to enable debug and grab a logcat while pressing the power button.

**Comment by samiraguiar** [comment]
With this new version, shortly after booting with the module enabled most buttons stop responding. But I now can see errors in the log viewer. Where are the log files stored?

**Comment by samiraguiar** [comment]
Just found it:

```
E/QC-QMI  : qmi_ctl_handle_request: qmi_ctl_tx_msg call failed
        Loading class com.spazedog.xposed.additionsgb.backend.XposedInjector
... <multiple times>
```

There were more messages available through the log viewer, but when I entered that screen the buttons stopped working so I couldn't send them.
One of them reads:

```
Error while injecting runtime code to the methods matching the name interceptKeyBeforeDispatching...
<stack trace>
Caused by: could not find any methods matching the name interceptKeyBeforeDispatching for the class com.android.internal.policy.impl.MiuiPhoneWindowManager.
```

**Comment by dbergloev** [comment]
Well it seams as if I have to fix a new issue before we can find the first one. I will look into this problem, which I already have an idea about. After that, hopefully we can track down your power button problem.

**Comment by dbergloev** [comment]
Try updating to version 3.6.1, it should work except for the initial issue with the power button. For that, I will still need some complete debug logs.

**Comment by samiraguiar** [comment]
Thanks for the fast fix.
With the new version my power button is working now. However, after I mapped it to some custom action it stopped responding.
I was going to send a log but the log viewer is empty, even with debug mode enabled. Any ideas?

**Comment by dbergloev** [comment]
In that case you need to get a log usin a logcat app or similar, while pressing whatever is not working

**Comment by samiraguiar** [comment]
I said it too early -- it's working now with version 3.6.1. I managed to remap the buttons and they're working fine. Thanks for the support!

**Comment by dbergloev** [comment]
Okay great
